ckanext/recombinant/templates/recombinant/snippets/choices_reference.html

```diff
@@ -1,4 +1,4 @@
-<h3>{{ _("Field choices reference") }}</h3>
```

Comment:
https://github.com/open-data/ckanext-canada/blob/master/ckanext/canada/i18n/en/LC_MESSAGES/ckan.po#L6374 has html in it. Maybe we should remove it here?